src/app/commands/poap/POAP.ts

```diff
@@ -11,19 +11,17 @@ import EarlyTermination from '../../errors/EarlyTermination';
 import EndPOAP from '../../service/poap/EndPOAP';
 import DistributePOAP from '../../service/poap/DistributePOAP';
 import ConfigPOAP from '../../service/poap/ConfigPOAP';
```

Comment:
By removing the `discordServerIds` import, do you have another idea of how to manage a phased rollout?

Reply:
I remember there was an issue with the commands not getting registered if the `guildIDs` wasn't present. Turns out that the bot has to be manually kicked and then reinvited for the commands to take affect. 

Regarding phased rollout, we'll be needing this and it'll have to include the 10 discord Ids. Let me put this restriction back

Reply:
Sounds good. Thank you, Brian 👍🙏